shared/android/app/src/main/java/io/keybase/ossifrage/MainActivity.java

```diff
@@ -449,4 +452,22 @@ public void setBackgroundColor(DarkModePreference pref) {
       mainWindow.setBackgroundDrawableResource(bgColor);
     });
   }
+
+  @Override
+  public boolean dispatchKeyEvent(KeyEvent event) {
```

Comment:
Does this not trigger off a virtual keyboard?

Reply:
@jzila doesn't look like it:

> You should never rely on receiving KeyEvents for any key on a soft input method. In particular, the default software keyboard will never send any key event to any application targetting Jelly Bean or later, and will only send events for some presses of the delete and return keys to applications targetting Ice Cream Sandwich or earlier. 

-- https://developer.android.com/reference/android/view/KeyEvent

Reply:
(would be super useful if you're able to test! gonna struggle to get a bluetooth keyboard connected to android here.)

Reply:
Ok I have one, let me build and test